ingestor/utils.js

```diff
@@ -0,0 +1,66 @@
+// TODO
+// await use axios post seedData fn to push data to db
+// done: delete file directory(or add to gitignore so zip files doesn’t push to Github) + drain
+
+const fs = require('fs');
+const path = require('path');
+const unzipper = require('unzipper'); // npm lib used to unzip files
+const { createReadStream } = require('fs');
+
+// async fs reads and unzip files to a directory
+
+const folderName = 'Archive.zip';
+const pathToExtract = 'unzippedArchive';
+
+const unzipFolder = (folderName, pathToExtract) => {
+  fs.createReadStream(`./${folderName}`).pipe(
+    unzipper.Extract({ path: __dirname + `/${pathToExtract}` })
+  );
+};
+
+class File {
+  constructor(filename) {
+    this.filename = filename;
+    this.fileContents;
+  }
+
+  // Read a file and save into the constructor
+  readFile() {
+    return new Promise((resolve, reject) => {
+      try {
+        fs.createReadStream(__dirname + '/channels.json')
```

Comment:
Why hard code `/channels.json`, I thought we have more than one json to read from?